aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,129 @@
+  AvatarAccessibilityExample,
+  AvatarDarkModeExample,
+  AvatarDataTableExample,
+  AvatarDataTableIconExample,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+  MarketplaceNavigationalCards,
+} from '../../examples';
+
+## When to use
+
+Avatars are used to enable users to quickly identify an object, whether that be a specific user, account, or other entity. Avatars are always paired with additional information, such as a user's name, for better clarity. If possible, use an image or logo over initials.
+
+The consistent dimensions of an avatar ensure that readability and scanability of content surrounding the avatar is not compromised. For example, when scanning a data table column that contains an avatar of a company’s logo and the company name, the company name stays aligned from one row to the next. 
+
+## When not to use
+Avatars are not the only components that provide visual representation. When applicable, use these over Avatars:
+- Use Icons when representing objects, actions, programs, or other communicative symbols.
+- Use an image when the context of the visual does not need to have equal height and width to maintain alignment of its surrounding content.
+
+## Anatomy
+
+Avatars can be displayed as an image, initials, or a logo. They come in three different sizes:
+
+| T-shirt size |  Dimensions (in pixels) |
+| ----------   | --------------|
+| **small**    |    24x24px    |
+| **medium**   |    48x48px    |
+| **large**    |    96x94px    |
+
+
+The container of an Avatar can be circular or a rounded square based on the use case.
+
+- Use a circular Avatar when representing a person by an image or their initials.
+- Use a rounded square Avatar when representing a company’s or entity’s logo. This can be achieved by applying `round='medium'` to the Avatar.
+- Always round the edges of a square container when using an avatar.
+
+<Example
+  figma="https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/avatar/AvatarSizePreviewExample.js"
+>
+  <AvatarSizePreviewExample />
+</Example>
+
+## Accessibility
+
+Inform users of assistive technologies what the avatar represents by including an `a11yTitle`. This text will be read out by a screen reader when the user reaches the avatar.
+
+If the avatar acts as a button, the `a11yTitle` should also inform the user of what action will occur if clicked.
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/avatar/AvatarAccessibilityExample.js"
+  width={{ min: 'medium' }}
+>
+  <AvatarAccessibilityExample />
+</Example>
+
+## Handling logos in dark mode
+When possible, provide both a light and dark mode version of a company logo. If a dark mode version of the logo is not available, apply background={{ color: ‘background’, dark: false }} to the avatar.
+
+<Example
```

Comment:
Can this example use a large-sized avatar instead?

Reply:
yes done